packaging/installer/netdata-updater.sh

```diff
@@ -144,7 +144,7 @@ update() {
   info "Current Version: ${current_version}"
   info "Latest Version: ${latest_version}"
 
-  if [ "${latest_version}" -gt 0 ] && [ "${current_version}" -gt 0 ] && [ "${current_version}" -ge "${current_version}" ]; then
+  if [ "${latest_version}" -gt 0 ] && [ "${current_version}" -gt 0 ] && [ "${latest_version}" -ge "${current_version}" ]; then
```

Comment:
please use `:-0` parameter expansion for `current_version`

Reply:
Please amend with default expansion for `${current_version:-0}` and I'll merge this

Reply:
Am I misreading something or isn't the check supposed to be reversed : `[ "${current_version}" -ge "${latest_version}" ]` ; ie. block the update if the current version >= latest version.

Reply:
yeah, good catch, you are right! 🙈

Reply:
yeah my bad 🤦‍♂️